tests/integration/test_use_cases/test_14_create_purchase.py

```diff
@@ -0,0 +1,27 @@
+"""
```

Comment:
I have been following the same numbers in the java integration tests in the devolperTC, so in theory this should be renamed to` test_17_asset_purchase.py`.

Also I'm using the informal naming convention `test_<number>_<object_name\s>_<operation>.py`

see: https://github.com/DEX-Company/starfish-java/blob/develop/src/test/java/sg/dex/starfish/integration/developerTC/PurchaseAsset_17.java